projects/igniteui-angular/src/lib/tabs/tabs.directive.ts

```diff
@@ -166,6 +172,26 @@ export abstract class IgxTabsDirective extends IgxCarouselComponentBase implemen
     protected scrollTabHeaderIntoView() {
     }
 
+    private setAttributes(igxName: string) {
+        this.items.forEach(item => {
+            this.setHeaderAttribute(item, 'aria-label', `${igxName}-tabitem-${NEXT_TAB_ID}`);

```

Comment:
We should not set the `aria-label`, but we need to set the `id` of the tab and tabpanel.

Reply:
Done